### PR TITLE
Alter filters for endpoints selection

### PR DESCRIPTION
**Pull request by JbannisterScottLogic**
Creating new filters to allow for pipeline to filter by endpoint

### CODE REVIEW

File: digital_land/pipeline.py [hunk 1]
```diff
@@ -194,13 +203,20 @@ def load_redirect_lookup(self):
             old_entity = row.get("old-entity", "")
             entity = row.get("entity", "")
             status = row.get("status", "")
-            if old_entity and status:
+            if old_entity and entity and status:
```
Comment (eveleighoj) [inline digital_land/pipeline.py:206]: is this on purpose? I think if the status is 410 then there doesn't need to be an entity

File: digital_land/pipeline.py [hunk 2]
```diff
@@ -194,13 +203,20 @@ def load_redirect_lookup(self):
...
                 self.redirect_lookup[old_entity] = {"entity": entity, "status": status}
 
-    def filters(self, resource=""):
-        d = self.filter.get("", {})
+    def filters(self, resource="", endpoints=""):
```
Comment (eveleighoj) [inline digital_land/pipeline.py]: the default endpoint here is a string but should it be a list?

File: digital_land/pipeline.py (hunk 1 above)
Comment (JbannisterScottLogic) [reply digital_land/pipeline.py]: Had some merge issues and was added in error. Corrected now

File: digital_land/pipeline.py (hunk 2 above)
Comment (JbannisterScottLogic) [reply digital_land/pipeline.py]: Line from an older commit. Filters now takes the list of endpoints accordingly

**Review by CarlosCoelhoSL — APPROVED** [review]
lgtm

**Pull request merged**